generator/src/main/java/org/corfudb/generator/Generator.java

```diff
@@ -66,7 +66,7 @@ public static void main(String[] args) {
             try {
                 appsFutures[x].get();
             } catch (Exception e) {
-                System.out.println("App Exception: " + e);
+                throw new RuntimeException(e);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00112)